modules/iam-assumable-role-with-oidc/main.tf

```diff
@@ -80,6 +82,16 @@ data "aws_iam_policy_document" "assume_role_with_oidc" {
           values   = var.oidc_fully_qualified_audiences
         }
       }
+
+      dynamic "condition" {
```

Comment:
replace:

```hcl
  dynamic "condition" {
        for_each = var.provider_conditions

        content {
          test     = condition.value.test
          values   = condition.value.values
          variable = condition.value.variable
        }
      }
```

Reply:
`policy_` is too generic - what policy is this referring to? This is only valid in the context of `var.provider_urls` or `var.provider_url` hence why I suggested `provider_conditions`. You could also do `provider_trust_policy_conditions` but just `policy_conditions` is too ambiguous for the current use

Reply:
Sure, renamed to `provider_trust_policy_conditions` 👍